provider/oracle/network/util_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package network_test
+
+import (
+	"github.com/juju/juju/provider/oracle/network"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+)
+
+type utilSuite struct{}
+
+var _ = gc.Suite(&utilSuite{})
+
+func (u *utilSuite) TestGetMacAndIP(c *gc.C) {
+	macAddress := "29-E7-9F-D9-83-C8"
+	ipAddress := "120.120.120.0"
+	address := []string{
+		macAddress,
+		ipAddress,
+	}
+
+	mac, ip, err := network.GetMacAndIP(address)
+	c.Assert(err, gc.IsNil)
+	c.Assert(mac, jc.DeepEquals, macAddress)
+	c.Assert(ip, jc.DeepEquals, ipAddress)
+}
+
+func (u *utilSuite) TestGetMacAndIPWithError(c *gc.C) {
```

Comment:
What about a call to verify: err = errors.Errorf("The address is not an mac neighter an ip %s", val)